flickit-assessment-core/src/test/java/org/flickit/assessment/core/application/port/in/subjectinsight/CreateSubjectInsightUseCaseParamTest.java

```diff
@@ -0,0 +1,64 @@
+package org.flickit.assessment.core.application.port.in.subjectinsight;
+
+import jakarta.validation.ConstraintViolationException;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.flickit.assessment.common.error.ErrorMessageKey.COMMON_CURRENT_USER_ID_NOT_NULL;
+import static org.flickit.assessment.core.common.ErrorMessageKey.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class CreateSubjectInsightUseCaseParamTest {
+
+    @Test
+    void CreateSubjectInsight_AssessmentIdIsNull_ErrorMessage() {
```

Comment:
change test names prefix from `CreateSubjectInsight` to `testCreateSubjectInsightParam`